apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -0,0 +1,402 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Tag
+    order: 1
+eleventyComputed:
+    sizes: "{% include './sizes.json' %}"
+shouldShowContents: true
+---
+
+## Overview
+
+The purpose of tags is to provide a quick and visually appealing way to highlight or identify specific items, topics or attributes. Tags enhance the user experience by simplifying content organisation and facilitating content discovery and filtering.
+Tags can be embedded in other components such as cards, data tables (among others) to help users understand and filter information more effectively.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/tag/overview.svg",
+    alt: "A group of tags that are horizontally stacked together."
```

Comment:
```suggestion
    alt: "A group of tags."
```

Reply:
Trying to find this is design - can't find it for the life of me XD if anyone can point me in the right direction that would be great